### ISSUE TITLE
ProjectPackages should allow subpackages

### ISSUE THREAD

**Issue by levinalex** [bugsnag/bugsnag-go#23]
right now `ProjectPackages` uses `filepath.Match` to check which parts of the stacktrace to filter:

```go
bugsnag.Configure(bugsnag.Configuration{
    ProjectPackages: []string{"main", "example.com/foo/*"},
}
```

However, wildcards (because of `Filepath.match`) will only be expanded into a single 'directory'. This matches `example.com/foo/bar` and `example.com/foo/baz` but *not* `example.com/foo/bar/fred` or `example.com/foo/bar/fred/bar`.

(my workaround to that was to say `{"example.com/foo/*", example.com/foo/*/*", example.com/foo/*/*/*"}` but that is brittle and ugly)

It would be nice if it was possible to match all subpackages explicitly (maybe via `"example.com/foo/**"`?)

I'd take a stab at implementing this if you are open to it

**Comment by snmaynard** [comment]
Hey @levinalex - this seems like it would be a solid addition to the library. If you want to take a stab at it that would be great. If not then someone will get around to this soon!

**Comment by kattrali** [comment]
Fixed by #25. Thanks!

**Comment by levinalex** [comment]
thank you :+1:
